gltfTutorial/gltfTutorial_005_MeshesTexturesImagesSamplers.md

```diff
@@ -0,0 +1,106 @@
+            "attributes": {
+              "POSITION": "accessorForPositions",
+              "NORMAL": "accessorForNormals",
+              "TEXCOORD_0": "accessorForTexCoords"
+            },
+            "indices": "accessorForIndices",
+            "mode": 4,
+            "material": "exampleMaterial"
+          }
+        ]
+    },
+    ...
+}
+```
+
+The `meshPrimitive` summarizes all information that describes how the respective part of the object will be rendered: It contains references to the geometry data in form of [`accessor`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-accessor) objects, as well as a reference to the [`material`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-material) that will be used for rendering. The details of the `accessor`s will be explained in the section about [Buffers, BufferViews and Accessors](gltfTutorial_007_BuffersBufferViewsAccessors.md). Here, only a simple, high-level description of accessors will be used, to give a basic understanding of how the geometry data is represented in the `meshPrimitive` objects of a glTF asset.
+
+#### Mesh primitive attributes
+
+In order to render the geometry of a `meshPrimitive`, the information about *what* should be rendered and *how* it should be rendered has to be combined. The information about *what* should be rendered is given by the actual geometry information. This is at least a set of vertices and certain attributes that belong to the vertices. The `attributes` dictionary of the `meshPrimitive` contains references to `acessor` objects. An `accessor` is an abstract source of data, and in the simplest case it can be imagined as a block of memory, together with some information about the data type and layout of this memory block. In the example above, the `attributes` contain references to three `accessors`. These accessors describe the positions and normal vectors for the vertices of the mesh, and the texture coordinates at these vertices:
+
+![meshPrimitive accessors](images/meshPrimitiveAttributes.png)    
+
+#### Mesh primitive mode  
+
+By default, the geometry data is assumed to describe a triangle mesh. Three consecutive elements of the accessors are interpreted as the attributes of the three vertices of a triangle. Different other rendering modes are possible: The geometry data may also describe individual points, lines or triangle strips. This is indicated by the `mode` that may be stored in the mesh primitive: Its value is an OpenGL constant like `GL_TRIANGLE_STRIP` or `GL_LINE_LOOP` that indicates how how the geometry data has to be interpreted.  
+
+#### Mesh primitive indices
+
+The mesh primitive can also contain an accessor with the key name `indices`. This accessor provides the indices of the vertices that the mesh elements consist of. For example, when the rendering `mode` is `GL_TRIANGLES`, then three consecutive elements of the indices accessor are the vertex indices of one triangle.

```

Comment:
This could use more detail and should also emphasize that indices are used in almost all cases.